ops/scripts/update-op-geth.py

```diff
@@ -11,34 +10,10 @@
 
 
 def main():
-	for project in ('op-service', 'op-node', 'op-proposer', 'op-batcher', 'op-bindings', 'op-chain-ops', 'op-e2e', 'op-wheel'):
+	for project in ('.', 'op-wheel'):
```

Comment:
Note that `indexer/go.mod` also references op-geth but wasn't previously included in the list to update.  Should it be added here?

Reply:
Yes, we should add the indexer here.

Reply:
Done.